external-resolver/src/main/scala/org/genivi/sota/resolver/vehicle/VehicleDAO.scala

```diff
@@ -29,4 +29,10 @@ object VehicleDAO {
   def list: DBIO[Seq[Vehicle]] =
     vehicles.result
 
+  def exists(vin: Vehicle.Vin): DBIO[Option[Vehicle]] =
```

Comment:
Only if you are a classical logician, I'm a constructivist -- exists provides the witness.